packages/mdc-textfield/README.md

```diff
@@ -142,32 +148,46 @@ See [here](../mdc-notched-outline/) for more information on using the outline su
 
-#### Helper Text
+### Disabled
+
+Add the `disabled` attribute to `<input>` if the `mdc-text-field` is disabled. You also need to add `mdc-text-field--disabled` to the `mdc-text-field`.
+
+```html
+<div class="mdc-text-field mdc-text-field--disabled">
+  <input type="text" id="disabled-text-field" class="mdc-text-field__input" disabled>
+  <label class="mdc-floating-label" for="disabled-text-field">Disabled text field</label>
+  <div class="mdc-line-ripple"></div>
+</div>
+```
+
+### Text Field with Helper Text
 
 The helper text provides supplemental information and/or validation messages to users. It appears on input field focus
 and disappears on input field blur by default, or it can be persistent.
 See [here](helper-text/) for more information on using helper text.
 
-#### Leading and Trailing Icons
+### Text Field with Leading and Trailing Icons
 
 Leading and trailing icons can be added within the box or outlined variants of MDC Text Field as visual indicators as
 well as interaction targets. See [here](icon/) for more information on using icons.
 
+## Style Customization
+
 ### CSS Classes
 
 CSS Class | Description
 --- | ---
-`mdc-text-field` | Mandatory
-`mdc-text-field--upgraded` | Indicates the text field is upgraded, normally by JavaScript
-`mdc-text-field--box` | Styles the text field as a box text field
-`mdc-text-field--outlined` | Styles the text field as an outlined text field
-`mdc-text-field--fullwidth` | Styles the text field as a full width text field
-`mdc-text-field--textarea` | Indicates the text field is a `<textarea>`
-`mdc-text-field--disabled` | Styles the text field as a disabled text field
-`mdc-text-field--dense` | Styles the text field as a dense text field
-`mdc-text-field--with-leading-icon` | Styles the text field as a text field with a leading icon
-`mdc-text-field--with-trailing-icon` | Styles the text field as a text field with a trailing icon
-`mdc-text-field--focused` | Styles the text field as a text field in focus
+`mdc-text-field` | Mandatory.
+`mdc-text-field--upgraded` | Indicates the text field is upgraded, normally by JavaScript.
+`mdc-text-field--box` | Styles the text field as a box text field.
+`mdc-text-field--outlined` | Styles the text field as an outlined text field.
+`mdc-text-field--fullwidth` | Styles the text field as a full width text field.
+`mdc-text-field--textarea` | Indicates the text field is a `<textarea>`.
+`mdc-text-field--disabled` | Styles the text field as a disabled text field.
+`mdc-text-field--dense` | Styles the text field as a dense text field.
+`mdc-text-field--with-leading-icon` | Styles the text field as a text field with a leading icon.
+`mdc-text-field--with-trailing-icon` | Styles the text field as a text field with a trailing icon.
+`mdc-text-field--focused` | Styles the text field as a text field in focus.
 
 ### Sass Mixins
 
```

Comment:
"Text Field Outline" -> "Outlined Text Field" in a couple of places

Reply:
* "outlined" is the modifier on the component
* "outline" in sass mixins should be referring to the Material property it controls, right? I think this is fine.